middle_end/flambda/lift_let_to_initialize_symbol.ml

```diff
@@ -277,7 +277,9 @@ let rec make_named (symbol, (path:access list)) : Flambda.named =
     Expr (
       Flambda.create_let block (make_named (symbol, t))
         (Flambda.create_let field
-           (Prim (Pfield (h, Pvalue Pgenval), [block], Debuginfo.none))
+           (* CR mshinwell: double-check this change *)
+           (Prim (Pfield (h, Pvalue Pgenval, Pointer, Mutable), [block],
+             Debuginfo.none))
```

Comment:
This seems safe: Values contained in the symbol can be anything.
But the read block is actually immutable.

Reply:
Fixed